CalibMuon/DTDigiSync/src/DTTTrigSyncFromDB.cc

```diff
@@ -36,15 +36,19 @@ DTTTrigSyncFromDB::DTTTrigSyncFromDB(const ParameterSet& config)
       theWirePropCorrType(config.getParameter<int>("wirePropCorrType")),
       // spacing of BX in ns
       theBXspace(config.getUntrackedParameter<double>("bxSpace", 25.)),
-      thetTrigLabel(config.getParameter<string>("tTrigLabel")) {}
+      thetTrigLabel(config.getParameter<string>("tTrigLabel")),
+      thet0Label("") {
+  if (config.exists("t0Label"))
```

Comment:
per Slava's suggestion, config.exists should be avoided. Can you find an alternative way to propagate the default? In EDProducers, it would be done via fillDescriptions, but I'm not sure what is the best way to do it in an arbitrary class.

Reply:
@battibass , you can add the "t0Label" to all existing configurations, not only the two you updated here

Reply:
@perrotta : yes, sorry (I will anyhow take action when we agree about how to proceed with the next point).
@jpata : do you have any suggestion that does not imply fillDescriptions? As the whole DT local reconstruction has this code nested at many levels, I guess the latter proposal would imply implement fillDescriptions for the whole DT local reconstruction architecture ...

Reply:
@battibass please proceed with adding the "t0Label" to all existing configurations. This will allow you getting rid of the `config.exists` in your code. 
Then for a possible (and welcome) overall migration to `fillDescriptions,` a separate PR can be better suited

Reply:
@perrotta : uhm, I am not sure this is the case, DT local reconstruction is also run @ HLT, shouldn't this be changed also in some HLT configuration before getting rid of `config.exists`?

Reply:
Perfect, then I proceed as you suggest, thanks!